templates/templates/preview/model.test

```diff
@@ -0,0 +1,112 @@
+
+== Models ==  
```

Comment:
If you need templates for unit tests, they shouldn't be in the set that is shipped. 

- Consider moving these to `/tests/test_configs` instead
- Consider using the same naming scheme
  - currently: `model.test` and `model.template`
  - maybe: `model_XXX.tpl` and `model_YYY.tpl`
  - I don't know the real difference for why each template is unique at a glance, but replace xxx/yyy w/purpose

Reply:
removed test templates (a mistake)